cmd/plasmacli/subcmd/keys/import.go

```diff
@@ -8,11 +8,13 @@ import (
 	"github.com/ethereum/go-ethereum/crypto"
 	"github.com/spf13/cobra"
 	"github.com/spf13/viper"
+	"io/ioutil"
 )
 
 // ImportCmd returns the keys import command
 func ImportCmd() *cobra.Command {
-	importCmd.Flags().String(fileF, "", "read the private key from the specified keyfile (must be absolute path)")
+	importCmd.Flags().String(fileF, "", "read the private key from raw private keyfile (must be absolute path)")
```

Comment:
Instead of having two flags for the two different types of files, we can reuse the `fileF` flag and simple have a boolean flag for if it's encrypted or not.

i.e `plasmacli keys import --file=key.txt --encrypted`. If `--encrypted` is not set, it assumes a raw private key file

(also as the early comment. Don't think this has to be an absolute path)

Reply:
I think `ioutil.ReadFile()` does require an absolute path, can adjust to use os.openfile() if desired? (readfile seems a bit cleaner though). Implementing the multiple flags now

Reply:
Does it required an aboslute path? The godoc doesn't mention that. https://godoc.org/io/ioutil#ReadFile

The impl of `ioutil.ReadFile` just calls `os.Open`.

Reply:
You're right - good catch